tracer/test/test-applications/debugger/dependency-libs/Samples.Probes.TestRuns/MetricOnMethodProbeTestDataAttribute.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+
+namespace Samples.Probes.TestRuns
+{
+    [AttributeUsage(AttributeTargets.Method | AttributeTargets.Constructor)]
+    public class MetricOnMethodProbeTestDataAttribute : MethodProbeTestDataAttribute
```

Comment:
```suggestion
    public class MetricMethodProbeTestDataAttribute : MethodProbeTestDataAttribute
```

Same reasoning as I gave for `MetricOnLineProbeTestDataAttribute`. Again, just nitpicking here.

Reply:
Done in ebb2b638668a8ffee58604d13ae658037d77b7d8.